### PR TITLE
Bug: delete all policies when management + hub hosted mode

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
description: When hcp clusters are created on the MCV2, policies are rolled out well.
When deleting the hcp cluster, we see that all the ACM policies on the ACM hub are also deleted.

solution: add  delete-orphan annotation into the policy crd
ref: https://issues.redhat.com/browse/ACM-6604

### CODE REVIEW

**Comment by yiraeChristineKim** [comment]
I don't know what is red-hat-trusted-app-pipeline
Red Hat Trusted App Pipeline Failing after 1s — Failed

**Comment by mprahl** [comment]
@yiraeChristineKim could you please change the commit title to reflect the fix and not the bug description?

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -48,6 +48,8 @@ spec:
           value: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
         - name: GOVERNANCE_POLICY_FRAMEWORK_ADDON_IMAGE
           value: quay.io/stolostron/governance-policy-framework-addon:latest
+        - name: HOSTING_CLUSTER_NAME
+          value: cluster1
```
Comment (mprahl) [inline config/manager/manager.yaml:52]: What would set this value? Also, a hosted cluster can have different hosting clusters than other hosted clusters on the same hub.

**Comment by mprahl** [comment]
@yiraeChristineKim could you please make this PR in the upstream?

**Comment by yiraeChristineKim** [comment]
> @yiraeChristineKim could you please make this PR in the upstream?

Oh you right I forgot that Thanks!

**Pull request closed without merging**